### PR TITLE
misc: fix icon alignment and migrate component to tailwind

### PR DESCRIPTION
**Pull request by ansmonjol**
This also fixes the loading state that was wrong

### CODE REVIEW

File: src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx [hunk 1]
```diff
@@ -111,14 +111,14 @@ const SubscriptionUsageLifetimeGraph = ({
         alignItems={'flex-start'}
         justifyContent={'space-between'}
       >
-        <Stack direction={'row'} gap={2} alignItems={'flex-end'}>
+        <div className="align flex flex-row items-center gap-2">
```
Comment (keellyp) [inline src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx:114]: what's align?

File: src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx [hunk 2]
```diff
@@ -177,22 +178,18 @@ const SubscriptionUsageLifetimeGraph = ({
                   loading={isLoading}
                 />
 
-                <GraphContainer $blur={isBlurred}>
-                  <GraphWrapper>
+                <div className={tw({ 'pointer-events-none blur-sm': isBlurred })}>
+                  <div className="flex flex-col gap-3">
+                    {/* {!!true ? ( */}
```
Comment (keellyp) [inline src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx:183]: if can be removed i guess

**Review by keellyp — APPROVED** [review]
👏

File: src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx (hunk 2 above)
Comment (ansmonjol) [reply src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx]: oupsi

File: src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx (hunk 1 above)
Comment (ansmonjol) [reply src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx]: Yeah took me tries to find the good one, this one does not exists actually

Does a linter exists to flag those non existing rules?

**Pull request merged**